packages/charting/src/components/ProgressBarChart/ProgressBarChart.base.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { IProcessedStyleSet } from '../../Styling';
+import { classNamesFunction } from '../../Utilities';
+import { IProgressBarChartProps, IProgressBarChartStyleProps, IProgressBarChartStyles } from './ProgressBarChart.types';
+
+const getClassNames = classNamesFunction<IProgressBarChartStyleProps, IProgressBarChartStyles>();
+
+export class ProgressBarChartBase extends React.Component<IProgressBarChartProps, {}> {
```

Comment:
Please rename anything that says ProgressBar to StackedChart or something more descriptive. We should not be adding Progress anything to the charting package. (There is already a ProgressIndicator and adding this here would cause confusion.)